types/chrome-location/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for chrome-location 1.2
+// Project: https://github.com/hughsk/chrome-location
+// Definitions by: Ingyu Tae <https://github.com/IngyuTae>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const exe: string;
```

Comment:
```suggestion
declare const exe: string | null;
```
this can be null, can't be?
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library